Sage/carbon
# chore(confirm): remove classic styling - FE-3104

### Proposed behaviour
Removal of classic theme styling from the Confirm component

### Current behaviour
Confirm component currently supports classic styling

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout remove_classic_theme_confirm`
- `npm start` to start Storybook
- Navigate to Confirm story in Storybook
- Confirm component should not have any classic styling
